drivers/wifi/nrf700x/osal/fw_if/umac_if/inc/fmac_api.h

```diff
@@ -1420,4 +1420,19 @@ enum wifi_nrf_status wifi_nrf_fmac_get_host_rpu_ps_ctrl_state(void *fmac_dev_ctx
 enum wifi_nrf_status wifi_nrf_fmac_set_listen_interval(void *fmac_dev_ctx,
 						       unsigned char if_idx,
 						       unsigned short listen_interval);
+
+/**
+ * @brief Configure WLAN ps to dtim or listen interval.
+ * @param[in] fmac_dev_ctx Pointer to the UMAC IF context for a RPU WLAN device.
+ * @param[in] if_idx Index of the interface on which power management is to be set.
+ * @param[in] ps_wake_interval Enable listen interval based ps(default dtim based)
```

Comment:
Description looks wrong, we now have a separate mode, so, this should just be interval, no?